### PR TITLE
[Trezor] Tx Signing Fixes

### PR DESCRIPTION
**Pull request by matheusd**
Some bugfixes for the current master for transaction signing when using trezor.

Individual commits should be easy to review individually.

### CODE REVIEW

File: app/helpers/msgTx.js [hunk 1]
```diff
@@ -182,84 +186,136 @@ export function decodeRawTransaction(rawTx) {
...
-  position += 1;
-  switch (first) {
-    case 0xfd:
-      tx.numInputs = rawTx.readUInt16LE(position);
-      position += 2;
-      break;
-    case 0xfe:
-      tx.numInputs = rawTx.readUInt32LE(position);
+
+  if (tx.serType != SERTYPE_ONLYWITNESS) {
```
Comment (victorgcramos) [inline app/helpers/msgTx.js:190]: I think we should use a strict comparison here instead of loose one.

**Review by victorgcramos — COMMENTED** [review]
Code looks good! Just found one minor detail, see my inline comment. Nice job (as always) 😄

File: app/helpers/msgTx.js (hunk 1 above)
Comment (matheusd) [reply app/helpers/msgTx.js]: Nice catch. Updated.

**Review by vctt94 — APPROVED** [review]
I tested on latest trezor-firmware master, commit: cd1d5195fd0f414c7bcb41dfda27fe2f1c4fd064. 
I noticed a bug when initializing trezor, but signing is working fine!

LGTM.

**Pull request merged**